src/lib.rs

```diff
@@ -581,29 +566,6 @@ mod test {
             ])
             .clean("<a rel=\"what\">s</a>");
     }
-    // The rest of these are stolen from
-    // https://code.google.com/p/html-sanitizer-testbed/source/browse/trunk/testcases/t10.html
-    #[test]
-    fn test_10() {
-        let fragment = "<SCRIPT/alert(10);/";
-        let result = clean(fragment);
-        assert_eq!(result, "");
-    }
-    #[test]
-    fn test_11() {
-        let fragment = "<IMG SRC=\"javas%63ript:alert(11);\">";
-        let result = clean(fragment);
-        assert_eq!(result, "");
-    }
-    #[test]
-    fn test_100() {
-        let fragment = "<!-- Here is a comment: -- This is a nested comment -->\n\
-                        <a href=\"http://harmless.com/This is still inside the comment: \
-                        --evadefilter><img onerror=alert(100) src=''/><a href=\"test\">\
-                        link</a>";
-        let result = clean(fragment);
-        assert_eq!(result, "\nlink");
-    }
```

Comment:
Are these no longer useful?

Reply:
They were never really useful. They tested the ability of the sanitizer to parse weird HTML corner cases, which means they're redundant to html5ever's own test suite. Using that linked test suite as designed requires you to involve a browser (the suite dosen't define what the resulting HTML should be, just that no JavaScript gets executed), which is really what we should do for that thing.